src/main.ts

```diff
@@ -20,27 +24,54 @@ async function run(): Promise<void> {
         if (currentVersion === null)
             return
 
-        const lastCommitMessage = await getLastCommitMessage()
-        if (lastCommitMessage === null)
-            return
-
-        const releaseType = getReleaseTypeFromCommitMessage(lastCommitMessage)
+        let releaseType: string | null = 'non-release'
```

Comment:
release type still can be null even with a default value?

Reply:
After some quick deliberation o slack, we agreed on:
```
 let releaseType: ReleaseType | null = null
```